lib/ts/recipe/emailpassword/index.ts

```diff
@@ -52,9 +52,10 @@ export default class Wrapper {
         });
     }
 
-    static createResetPasswordToken(userId: string, userContext?: any) {
+    static createResetPasswordToken(userId: string, email: string, userContext?: any) {
```

Comment:
Add a comment here explaining that we do not make email optional here cause we want to allow passing in pf primary user ID. If we make email optional, and if the user provides a primary user ID, then it may result in two problems:
- there is no recipeUserId = input primary user ID, in this case, this function will throw an error
- There is a recipe userId = input primary user ID, but that recipe has no email, or has wrong email compared to what the user wanted to generate a reset token for.

And we want to allow primary user ID being passed in.

Also make an ADR for this.